pages/posts/roadmap-2024.md

```diff
@@ -0,0 +1,228 @@
+    - Beacon network (alpha)
+    - Glados network monitoring tool
+    - Testing framework (Portal Hive)
+- Conference talks:
+    - \> 5 talks dedicated to Portal Network were delivered at major ETH events in 2023
+- Team growth:
+    - several new team members have joined in 2023
+- Collaborations:
+    - many new connections with app and client teams
+- Communications:
+    - new website and blog
+
+This activity has set the stage for 2024 to be a pivotal year for Portal Network as some key sub-protocols move from alpha into production.
+
+As well as successes, there were some frustrations for the Portal Network team in 2023:
+
+
+## 2024 workstreams
+
+To achieve our mission, we’re focused on 5 major workstreams that comprise our roadmap.
+
+### 1) Trin
+
+- The Trin team comprises 5 developers.
+- Trin developers also maintain tooling such as Glados, Portal-Hive, and Portal-Bridge.
+- The Trin team is especially focused on delivering a stable 4444s History network with validation for post-merge data via the Beacon network.
+- Specific challenges they might encounter in delivering the 2024 roadmap include...
+  - Stabilizing `utp` transport tooling / debugging async network issues
+  - Stabilizing the History/Beacon networks, while supporting the alpha State network
+  - Scaling pains of monitoring tools
+  - Network failures due to unmitigated attack vectors
+ 
+### 2) Fluffy
+
+- The Fluffy team comprises 2 developers.
+- Fluffy developers also assist on tooling such as Glados and Portal-Hive.
+- The Fluffy team is especially focused on delivering a stable Portal history network and further work on the specifications and implementation of the Portal beacon network which allows for validation of post-merge data.
+- The Fluffy team is also focused on developing a PoC that integrates the Portal network with an EL client (Nimbus).
+- Specific challenges they might encounter in delivering the 2024 roadmap include...
+  - Stabilizing the History/Beacon networks, while supporting R&D on the Portal state network.
+  - Balancing time and efforts to further develop the Fluffy version of a Portal-bridge and actively run it on the network.
+  - Client/network robustness, resilience and performance as the networks scale up in terms of amounts of nodes and data + the capability to properly monitor this. 
+  
+
+### 3) Ultralight
+
+- The Ultralight team comprises N developers.
+- Ultralight developers also work on tooling such as Glados and maintaining the testing infrastructure.
+- The Ultralight team is especially focused on....
+- Specific challenges they might encounter in delivering the 2024 roadmap include...
+...
+### 4) Glados
+
+- Glados is worked on by developers from the three main client teams, with some individuals taking more prominent roles than others. 
+- Scaling is going to be a challenge for Glados in 2024. The tool may have to migrate to an EF managed server so we can benefit from devops support and handle large volumes of traffic.
```

Comment:
Glados currently runs on an EF managed server, specifically managed by Paul on the devops team. Not sure if this is referring to something else. 
```suggestion
- Scaling Glados from one to three Portal networks while simultaneously scaling up the expected traffic volume will be a challenge in 2024. The primary bottleneck in this scaling will be the Postgres DB that holds all of Glados's data. The primary path to accomplishing this scaling will be to continue improving Glados's DB queries and indexing. 
```